performance/plugins/testRunner.ts

```diff
@@ -263,3 +153,16 @@ export async function runQuery(
   testLog.getLog().info("All promises fulfilled");
   return results;
 }
+
+async function runTxsInBurstMode(
+  mgaNodeandUsers: Map<
+    number,
+    { mgaSdk: Mangata; users: { nonce: BN; keyPair: KeyringPair }[] }
+  >,
+  testParams: TestParams,
+  generator: any
+) {
+  testParams.pending = Number.MAX_VALUE;
+  testParams.totalTx = Math.min(testParams.totalTx, 10000);
+  return runTxsInConcurrentMode(mgaNodeandUsers, testParams, generator);
```

Comment:
reuse concurrent mode to achieve burst functionality by ignoring pending parmeter